tests/test_icml_conference.py

```diff
@@ -1309,3 +1328,114 @@ def test_decision_stage(self, openreview_client, helpers):
+            invitations='ICML.cc/2023/Conference/-/Edit',
+            readers = ['ICML.cc/2023/Conference'],
+            writers = ['ICML.cc/2023/Conference'],
+            signatures = ['ICML.cc/2023/Conference'],
+            invitation = openreview.api.Invitation(
+                id = 'ICML.cc/2023/Conference/-/Submission',
+                reply_forum_views = [
+                    {
+                        "id": "all",
+                        "label": "All"
+                    },
+                    {
+                        "id": "reviewers-chat",
+                        "label": "Reviewers Chat",
+                        "filter": "invitation:ICML.cc/2023/Conference/Submission${note.number}/-/Chat,ICML.cc/2023/Conference/Submission${note.number}/-/Official_Review",
+                        "nesting": 2,
+                        "sort": "date-asc",
+                        "layout": "chat",
+                        "live": True,
+                        "expandedInvitations": ["invitation:ICML.cc/2023/Conference/Submission${note.number}/-/Chat"]
+                    }
+                ]
+            )
+        )
+
+        submission_invitation = openreview_client.get_invitation('ICML.cc/2023/Conference/-/Submission')
+        assert len(submission_invitation.reply_forum_views)
+
+        submission = openreview_client.get_notes(invitation='ICML.cc/2023/Conference/-/Submission', number=1)[0]
+
+        openreview_client.post_invitation_edit(
+            invitations='ICML.cc/2023/Conference/-/Edit',
+            readers = ['ICML.cc/2023/Conference'],
+            writers = ['ICML.cc/2023/Conference'],
+            signatures = ['ICML.cc/2023/Conference'],
+            invitation = openreview.api.Invitation(
+                id = 'ICML.cc/2023/Conference/Submission1/-/Chat',
+                readers = ['everyone'],
+                writers = ['ICML.cc/2023/Conference'],
+                signatures = ['ICML.cc/2023/Conference'],
+                invitees = ['ICML.cc/2023/Conference/Program_Chairs', 'ICML.cc/2023/Conference/Submission1/Reviewers'],
+                edit = {
+                    'readers': ['ICML.cc/2023/Conference', '${2/signatures}'],
+                    'writers': ['ICML.cc/2023/Conference'],
+                    'signatures': {
+                        'param': {
+                            'enum': [
+                                'ICML.cc/2023/Conference/Program_Chairs',
+                                'ICML.cc/2023/Conference/Submission1/Reviewer_.*'
+                            ]
+                        }
+                    },
+                    'note': {
+                        'readers': ['ICML.cc/2023/Conference/Program_Chairs', 'ICML.cc/2023/Conference/Submission1/Reviewers'],
+                        'writers': ['ICML.cc/2023/Conference'],
+                        'signatures': ['${3/signatures}'],
+                        'forum': submission.id,
+                        "replyto": {
```

Comment:
Does this mean that the replyto field of the note will always be set to the forum id? I was thinking that users would still be able to reply to chat messages, even though they wouldn't appear nested like in the regular view.

Reply:
replyto is a param and it means that any value (note id) must have a forum that matches with the submisison.id.

This is exactly how it works with comment invitations.

Reply:
I see